src/Data/Contract/ContractDataClass.ts

```diff
@@ -0,0 +1,6 @@
+import { localContractDataClass } from './LocalStorage/localContractDataClass'
+import { pisaContractDataClass } from './Pisa/pisaContractDataClass'
+
+export const ContractPromise = import.meta.env.ENABLE_PISA
```

Comment:
Please also implement `contract-document` similar to `quote-document` and add it to the details view of the contract (see https://github.com/pisasales/server-psa/issues/50#issuecomment-2175965907).